lib/process-services/src/lib/process-list/components/start-process.component.scss

```diff
@@ -1,54 +1,61 @@
-.adf {
-    &-start-process {
-        width: 96%;
-        margin-left: auto;
-        margin-right: auto;
-        margin-top: 10px;
-        .mat-select-trigger {
-            font-size: 14px !important;
-        }
-        mat-form-field {
-            width: 100%;
-        }
-        mat-select {
-            width: 100%;
-            padding: 16px 0 0;
-        }
-        .mat-content-actions {
-            text-align: right;
-        }
-        .mat-button {
-            text-transform: uppercase !important;
-        }
-    }
-    &-process-input-container {
-        mat-form-field {
-            width: 100%;
+@mixin adf-process-services-create-theme($theme) {
+    .adf {
+        &-start-process {
+            width: 96%;
+            margin-left: auto;
+            margin-right: auto;
+            margin-top: 10px;
+            .mat-select-trigger {
+                font-size: 14px !important;
+            }
+            mat-form-field {
+                width: 100%;
+            }
+            mat-select {
+                width: 100%;
+                padding: 16px 0 0;
+            }
+            .mat-form-field-label {
+                color: map_get($mat-grey,A400);
```

Comment:
I think this is not correct, we are using "map-get" in other files, see 
https://github.com/Alfresco/alfresco-ng2-components/blob/development/lib/content-services/src/lib/breadcrumb/breadcrumb.component.scss#L18